### ISSUE TITLE
[Chip] In Safari (iOS) , Focus outline does not hug component

### ISSUE THREAD

**Issue by macandcheese** [Esri/calcite-design-system#6976]
### Actual Behavior

In Safari on iOS - the focus outline does not hug the Chip - we need to add style to achieve this:

Safari on iOS:
<img width="575" alt="Screen Shot 2023-05-17 at 11 49 42 AM" src="https://github.com/Esri/calcite-components/assets/4733155/a59a5a10-3af4-409d-b054-de65a63a8657">

Safari on macOS (same in Chrome, Firefox):
<img width="861" alt="Screen Shot 2023-05-17 at 11 57 28 AM" src="https://github.com/Esri/calcite-components/assets/4733155/76a3bf27-4e16-4a47-98b7-2afd7449a51a">

### Expected Behavior

The focus outline should be rounded to hug the Chip.

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/chip-group/

### Reproduction Steps

1. View the sample on Developer Site in Simulator or on an iOS device.
2. Notice the focus outline is not hugging the Chip.

### Reproduction Version

1.3.1

### Relevant Info

Related: https://github.com/Esri/calcite-components/issues/4633

### Priority impact

p3 - want for upcoming milestone

### Esri team

Calcite (design)

**Comment by DitwanP** [comment]
This looks to have been resolved and was maybe a Safari related bug because when trying to repro on my phone I don't see the same behavior on the listed repro version. 

I found a bug report that might have been related but I'm not 100% sure.
[Bug report 1](https://bugs.webkit.org/show_bug.cgi?id=20807)

Safari on iOS live site:
![Chip Group  Calcite Design System  Esri Developer](https://github.com/user-attachments/assets/3d2f7182-bc5f-4ed9-8ea0-b1008c005a15)

Codepen on repro version and Safari: 
![image](https://github.com/user-attachments/assets/59017f5e-70d9-4cee-adf7-072dca3a45ca)
